core/templates/components/forms/custom-forms-directives/image-uploader.component.ts

```diff
@@ -95,9 +96,48 @@ export class ImageUploaderComponent {
     let filename: string = this.imageInputRef.nativeElement.value.split(
       /(\\|\/)/g).pop();
     this.errorMessage = this.validateUploadedFile(file, filename);
+    let fchng = this.fileChanged;
+    let newFile: File = new File(['asd'], 'so.txt', { type: 'text/plain' });
```

Comment:
newFile is only used much later and the scope is limited. Can we declare this over there?